src/main/java/org/demo/service/cxbox/inner/ClientReadWriteService.java

```diff
@@ -103,29 +111,31 @@ public Actions<ClientWriteDTO> getActions() {
 		return Actions.<ClientWriteDTO>builder()
 				.save(sv -> sv)
 				.action(act -> act
-								.scope(ActionScope.RECORD)
```

Comment:
pls, change only code corresponding to jira task. (diff is used to update projects - formatting changes in old code should be done in separate task if they do not break CI/CD)